ios/default.nix

```diff
@@ -292,9 +293,12 @@ nixpkgs.runCommand "${executableName}-app" (rec {
   cp --no-preserve=mode "$runInSim" "$out/bin/run-in-sim"
   chmod +x "$out/bin/run-in-sim"
   ln -s "$exePath/bin/${executableName}" "$out/${executableName}.app/"
-  cp -RL "${staticSrc}"/* "$out/${executableName}.app/"
-  for icon in "${staticSrc}"/assets/Icon*.png; do
+  cp -RL '${staticSrc}'/* "$out/${executableName}.app/"
+  for icon in '${staticSrc}'/assets/Icon*.png; do
     cp -RL "$icon" "$out/${executableName}.app/"
   done
+  for splash in '${staticSrc}'/assets/Default*.png; do
```

Comment:
is there any documentation we can link to about the expected/canonical file hierarchy?

Reply:
I'm not actually sure if this works yet... And if it does it seems kinda arbitrary. So yeah definitely need to at least document this or better make it more formal.